### PR TITLE
fix/ios push notifications

### PR DESCRIPTION
**Pull request by maycon-mello**
dev

### CODE REVIEW

File: ios/DockApp.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -789,9 +791,11 @@
 				CLANG_ENABLE_MODULES = YES;
 				CODE_SIGN_ENTITLEMENTS = DockApp/DockApp.entitlements;
 				CODE_SIGN_IDENTITY = "iPhone Distribution";
+				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
 				CODE_SIGN_STYLE = Manual;
-				CURRENT_PROJECT_VERSION = 396;
+				CURRENT_PROJECT_VERSION = 397;
```
Comment (mike-parkhill) [inline ios/DockApp.xcodeproj/project.pbxproj:796]: version code jumped to 400 above, should they match?

File: ios/DockApp.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -819,11 +824,12 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_ENABLE_MODULES = YES;
-				CODE_SIGN_ENTITLEMENTS = DockApp/DockApp.entitlements;
+				CODE_SIGN_ENTITLEMENTS = DockApp/DockAppRelease.entitlements;
 				CODE_SIGN_IDENTITY = "iPhone Distribution";
 				CODE_SIGN_STYLE = Manual;
-				CURRENT_PROJECT_VERSION = 396;
+				CURRENT_PROJECT_VERSION = 397;
```
Comment (mike-parkhill) [inline ios/DockApp.xcodeproj/project.pbxproj:830]: here too

File: ios/DockApp.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -819,11 +824,12 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_ENABLE_MODULES = YES;
-				CODE_SIGN_ENTITLEMENTS = DockApp/DockApp.entitlements;
+				CODE_SIGN_ENTITLEMENTS = DockApp/DockAppRelease.entitlements;
```
Comment (mike-parkhill) [inline ios/DockApp.xcodeproj/project.pbxproj:827]: this probably needs to be configurable for white label apps, right?

File: ios/DockApp.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -839,6 +845,7 @@
 				PRODUCT_BUNDLE_IDENTIFIER = labs.dock.DockApp;
 				PRODUCT_NAME = DockApp;
 				PROVISIONING_PROFILE_SPECIFIER = "match AppStore labs.dock.DockApp";
+				"PROVISIONING_PROFILE_SPECIFIER[sdk=iphoneos*]" = "match AppStore labs.dock.DockApp";
```
Comment (mike-parkhill) [inline ios/DockApp.xcodeproj/project.pbxproj:848]: are we updating these for white label?

File: ios/DockApp.xcodeproj/project.pbxproj (hunk 1 above)
Comment (maycon-mello) [reply ios/DockApp.xcodeproj/project.pbxproj:796]: Yes, let me check this

**Review by mike-parkhill — APPROVED** [review]

**Pull request merged**